platform/docs/docs/platform/services/data/HangingProtocolService.md

```diff
@@ -65,7 +70,13 @@ There are two events that get publish in `HangingProtocolService`:
 
 Default initialization of the modes handles running the `HangingProtocolService`
 
+## Hanging Protocol Instance Definition
+A hanging protocol has an id and/or a name as provided by the extension, and
```

Comment:
This is confusing and unnecessary to me, why we need to have `name` as well as `id`?